### PR TITLE
fix: move breathing bar above thirst bar when Thirst is active 

### PR DESCRIPTION
**Pull request by mohamedation**
added Breathing as optional dependency and created an override on Breathing bar position so it doesn't collide with Thirst bar.

### CODE REVIEW

File: overrides/Breathing/ui/Breathing.ui [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          "position-horizontal-center": {},
+          "position-bottom": {
+            "target": "BOTTOM",
+            "offset": 102
+          }
+        }
+      }
+    ]
+  }
+}
```
Comment (Cervator) [inline overrides/Breathing/ui/Breathing.ui:28]: Trivial best practice: end all text type files with an empty newline :-)

Not something worth changing here, just FYI!

**Review by Cervator — APPROVED** [review]
Looks fine to me, although I haven't tested - including a screenshot of it in action would help prove that easily :-)

File: overrides/Breathing/ui/Breathing.ui (hunk 1 above)
Comment (mohamedation) [reply overrides/Breathing/ui/Breathing.ui:28]: > Trivial best practice: end all text type files with an empty newline :-)
> 
> Not something worth changing here, just FYI!

thank you! i will follow that from now on

**Comment by mohamedation** [comment]
> Looks fine to me, although I haven't tested - including a screenshot of it in action would help prove that easily :-)
Screenshot from 2021-10-30 00-19-46

**Review by jdrueckert — APPROVED** [review]
Tests out nicely :+1:

**Pull request merged**